fe/fe-core/src/main/java/com/starrocks/planner/OlapTableSink.java

```diff
@@ -451,6 +451,8 @@ private void setIndexAndBucketNums(Partition partition, TOlapTablePartition tPar
             tPartition.addToIndexes(new TOlapTableIndexTablets(index.getId(), Lists.newArrayList(
                     index.getTablets().stream().map(Tablet::getId).collect(Collectors.toList()))));
             tPartition.setNum_buckets(index.getTablets().size());
+            long targetPartitionId = index.getTargetPartitionId() != 0 ? index.getTargetPartitionId() : partition.getId();
+            tPartition.addToAssociated_partition_ids(targetPartitionId);
```

Comment:
If we suppose that base table and target table have same partitions, then we can get the target table's PartitionID by target table and partition name?